index.js

```diff
@@ -44,10 +44,12 @@ module.exports = function configParser(yaml, templateFactory) {
         .then(phaseGeneratePermutations)
         // Output in the right format
         .then(doc => ({
+            annotations: Hoek.reach(doc, 'annotations', { default: {} }),
```

Comment:
@r3rastogi  According to @FenrirUnbound, you should be able to manually alter the table to have the field. I'm not a fan of having code that is not doing anything. It's a little confusing. I think we should either alter the table, or remove code that is not doing anything.